### PR TITLE
utils.views: Fix missing Content-Type handling

### PR DESCRIPTION
**Pull request by Jakub89**

### CODE REVIEW

File: concent_api/utils/tests/test_integration_api_view.py [hunk 1]
```diff
@@ -185,3 +185,43 @@ def test_provider_forces_computed_task_report_empty_key_returns_400_error(self):
...
+        serialized_task_to_compute      = dump(task_to_compute,             PROVIDER_PRIVATE_KEY,   REQUESTOR_PUBLIC_KEY)
+        deserialized_task_to_compute    = load(serialized_task_to_compute,  REQUESTOR_PRIVATE_KEY,  PROVIDER_PUBLIC_KEY, check_time = False)
+
+        force_report_computed_task = message.ForceReportComputedTask(
+            timestamp = int(dateutil.parser.parse("2017-12-01 10:59:00").timestamp()),
+        )
+        force_report_computed_task.task_to_compute  = deserialized_task_to_compute
+        serialized_force_report_computed_task       = dump(force_report_computed_task, PROVIDER_PRIVATE_KEY, CONCENT_PUBLIC_KEY)
+
+        with freeze_time("2017-12-01 10:59:00"):
```
Comment (rwrzesien) [inline concent_api/utils/tests/test_integration_api_view.py]: There is no need to use freeze_time in this test.

File: concent_api/utils/api_view.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def wrapper(request, *args, **kwargs):
             message = None
         else:
             if request.content_type == '':
-                raise Http400('Content-Type is missing.')
+                return JsonResponse({'error': 'Content-Type is missing.'}, status = 400)
```
Comment (rwrzesien) [inline concent_api/utils/api_view.py]: There is a check above
```
if request.content_type not in ['application/octet-stream', '']:
             return JsonResponse({'error': 'Concent supports only application/octet-stream.'}, status = 415)
```
and below
```
else:
                 return JsonResponse({'error': "Concent supports only application/octet-stream."}, status = 400)
```
It looks like double checking the same thing, but with different status response. Please unify that.

**Comment by Jakub89** [comment]
@rwrzesien I've made some adjustment.

**Comment by Jakub89** [comment]
Done.

**Pull request closed without merging**